app/src/main/java/com/mensinator/app/business/ClueImport.kt

```diff
@@ -0,0 +1,106 @@
+package com.mensinator.app.business
+
+import android.content.Context
+import android.widget.Toast
+import org.json.JSONArray
+import java.io.BufferedReader
+import java.io.File
+import java.io.InputStreamReader
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import androidx.core.database.sqlite.transaction
+import java.io.FileInputStream
+
+class ClueImport(
+    private val context: Context,
+    private val dbHelper: IPeriodDatabaseHelper,
+) : IClueImport {
+
+    override fun getDefaultImportFilePath(): String {
+        return File(context.getExternalFilesDir(null), "import.json").absolutePath
+    }
+
+    override fun importFileToDatabase(filePath: String) {
+        val db = dbHelper.writableDb
+
+        // Read JSON data from the file
+        val file = File(filePath)
+        val fileInputStream = FileInputStream(file)
+        val reader = BufferedReader(InputStreamReader(fileInputStream))
+
+        Toast.makeText(context, "Importing file...", Toast.LENGTH_SHORT).show()
+
+        val stringBuilder = StringBuilder()
+        var line: String? = reader.readLine()
+        while (line != null) {
+            stringBuilder.append(line)
+            line = reader.readLine()
+        }
+        reader.close()
```

Comment:
Apparently this can be replaced by 

`val fileContent = reader.use { it.readText() }`, but please check whether it works